CustomKnight/Skin/Base/SelectableSkin.cs

```diff
@@ -74,12 +96,18 @@ public class StaticSkin : ISelectableSkin{
         public string GetId() => SkinDirectory;
         public string GetName() => SkinDirectory;
         public bool hasSwapper() => true;
+        public bool hasInv() => DirectoryExists($"{SkinManager.SKINS_FOLDER}/{SkinDirectory}/Inventory");
         public string getSwapperPath() => Path.Combine(SkinManager.SKINS_FOLDER,SkinDirectory);
 
         public bool Exists(string FileName){
             string file = ($"{SkinManager.SKINS_FOLDER}/{SkinDirectory}/{FileName}").Replace("\\", "/");
             return File.Exists(file);
         }
+        public bool InvExists(string FileName)
```

Comment:
can the exists function not work for this ? by passing "Inventory/{FileName}" as it's param?